daphne_worker_test/wrangler.toml

```diff
@@ -3,7 +3,7 @@ main = "build/worker/shim.mjs"
 compatibility_date = "2022-10-14"
 
 [build]
-command = "cargo install --git https://github.com/cloudflare/workers-rs && worker-build"
+command = "cargo install --git https://github.com/cloudflare/workers-rs --version 0.0.8 worker-build && worker-build"
```

Comment:
What's the reasoning for this change?

Reply:
it compiles only worker-build with specific version since workers-rs is a set of crates.
and does the same in the Dockerfile. 
By doing so, worker-build is fetched and compiled once.
But these are local changes, and they are not related to this pr.

Reply:
In that case, could you revert? (1) I'd like to keep this change minimal, and (2) in the past we have needed to track particular commits in workers-rs, and when we do this, we need to make sure to build the same version of worker-build.